src/olympia/yara/models.py

```diff
@@ -28,4 +28,4 @@ def add_match(self, rule, tags=None, meta=None):
 
     @property
     def matched_rules(self):
-        return [match['rule'] for match in self.matches]
+        return sorted({match['rule'] for match in self.matches})
```

Comment:
It doesn't really make sense to use a `set` here instead of a list comprehension. Actually, you can just use a generator instead too `sorted((match['rule'] for match in self.matches))` - although I think using a list comprehension here might even be the "fastest" (not that we care…) since it doesn't need to be computed on every iteration compared to a generator or a set (where we have the unique checks every time).